app/src/main/res/values/strings.xml

```diff
@@ -1197,12 +1197,14 @@
     <string name="logins_doorhanger_save_confirmation">Save</string>
     <!-- Negative confirmation that Fenix should not save the new or updated login -->
     <string name="logins_doorhanger_save_dont_save">Don’t save</string>
+    <!-- Shown in snackbar to tell user that an item has been copied -->
+    <string name="login_copied">Copied to clipboard</string>
     <!-- Shown in snackbar to tell user that the password has been copied -->
     <string name="logins_password_copied">Password copied to clipboard</string>
     <!-- Shown in snackbar to tell user that the username has been copied -->
     <string name="logins_username_copied">Username copied to clipboard</string>
-    <!-- Shown in snackbar to tell user that the site has been copied -->
-    <string name="logins_site_copied">Site copied to clipboard</string>
+    <!-- Shown in snackbar to tell user that the web address has been copied -->
+    <string name="logins_web_address_copied">Site copied to clipboard</string>
```

Comment:
Can we revert this change? I'm not sure if we can change the IDs of strings without re-translating them but we can also check with l10n :/

Reply:
good point